backend/server/websocket_manager.py

```
@@ -92,24 +93,25 @@ async def disconnect(self, websocket: WebSocket):
                 await websocket.close()
-            except:
+            except Exception:
                 pass  # If this fails too, there's nothing more we can do
 
-    async def start_streaming(self, task, report_type, report_source, source_urls, document_urls, tone, websocket, headers=None, query_domains=[], mcp_enabled=False, mcp_strategy="fast", mcp_configs=[]):
+    async def start_streaming(self, task, report_type, report_source, source_urls, document_urls, tone, websocket, headers=None, query_domains=[], mcp_enabled=False, mcp_strategy="fast", mcp_configs=[], max_search_results=None):
         """Start streaming the output."""
         tone = Tone[tone]
         # add customized JSON config file path here
-        config_path = "default"
+        config_path = os.environ.get("CONFIG_PATH", "default")
```

Ruff (0.15.5)

[error] Undefined name `os`

(F821)